src/Connection/Client.php

```diff
@@ -8,17 +8,38 @@
 use Bunny\ClientStateEnum;
 use Bunny\Exception\BunnyException;
 use Bunny\Exception\ClientException;
+use Bunny\Protocol\AbstractFrame;
 use Bunny\Protocol\HeartbeatFrame;
+use Nette\Utils\Strings;
 
+/**
+ * @codeCoverageIgnore
+ */
 class Client extends BunnyClient
 {
+	/**
+	 * Constructor.
+	 *
+	 * @param array<string|mixed> $options
+	 */
+	public function __construct(array $options = [])
+	{
+		// Construct parent
+		parent::__construct($options);
+		$this->options['cycle_callback'] = $options['cycle_callback'];
```

Comment:
I see now that there is already naming convention on callbacks such as 'onFrameRecieved' so maybe `onRun` would be good naming

Reply:
Yeah, but there is heartbeat_callback field in options, so not sure what naming conventions we should use